assets/sass/_cookie_control.scss

```diff
@@ -389,4 +389,67 @@
     &[open] {
         opacity: 1;
     }
+
+    // cookie policy addition
+    .ng-cc-cookie-policy-text {
+        position: absolute;
+        inset: 0;
+        z-index: 10000000;
```

Comment:
100000 was not enough? :D 

please add a proper value based on the bootstrap z-index states:
https://getbootstrap.com/docs/5.2/layout/z-index/